src/app/workspace/resource/representation/still-image/still-image.component.ts

```diff
@@ -91,12 +114,14 @@ export class StillImageComponent implements OnChanges, OnDestroy {
 
     @Output() regionClicked = new EventEmitter<string>();
 
+    private _regionDrawMode: Boolean = false; // stores whether viewer is currently drawing a region
+    private _regionDragInfo; // stores the information of the first click for drawing a region
+    private _mouseTracker; // stores the MouseTracker that allows drawing regions
```

Comment:
is this used anywhere? My IDE says it's unused so it can probably be removed

Reply:
At some point this got the whole thing running, but apparently it is not needed anymore. I have removed it.